discord/ext/commands/core.py

```diff
@@ -704,14 +764,14 @@ async def call_before_hooks(self, ctx):
         if hook is not None:
             await hook(ctx)
 
-    async def call_after_hooks(self, ctx):
+    async def call_after_hooks(self, ctx: Context) -> None:
         cog = self.cog
         if self._after_invoke is not None:
             instance = getattr(self._after_invoke, '__self__', cog)
             if instance:
-                    await self._after_invoke(instance, ctx)
+                    await self._after_invoke(instance, ctx)  # type: ignore
```

Comment:
Why is this being ignored?

Reply:
type-checker believes argument could still be none, would need to cast.